packages/psss-server/src/routes/reports/helpers.ts

```diff
@@ -0,0 +1,12 @@
+/**
+ * Tupaia
+ * Copyright (c) 2017 - 2021 Beyond Essential Systems Pty Ltd
+ */
+
+const SYNDROMES: any[] = ['AFR', 'DIA', 'DLI', 'ILI', 'PF'];
```

Comment:
Thoughts on moving this to `/constants.ts`?

Reply:
Also, not `string[]`? `any` is evil 😝

Reply:
Even better, string literal? eg. `SYNDROMES: ['AFR', 'DIA', 'DLI', 'ILI', 'PF'] = ['AFR', 'DIA', 'DLI', 'ILI', 'PF'];` a bit of repetition but allows for some cool ts magic 😂

Reply:
Good point! I agree that `any` is evil 😛 I guess I got a bit lazy doing correct TS checking here, will fix this